src/OrchardCore.Modules/OrchardCore.AdminTrees/Views/Node/Edit.cshtml

```diff
@@ -11,6 +11,12 @@
     <input type="hidden" asp-for="AdminNodeType" />
 
     <fieldset>
+        <div class="form-group">
+            <label asp-for="SelectionPriority">@T["Selection Priority"]</label>
+            <input asp-for="SelectionPriority" class="form-control col-md-2" >
+            <span class="hint">@T["There can be only one item selected at the same time. In case there are more than one competing the system will use this value to determine which one wins."]</span>
```

Comment:
```suggestion
            <span class="hint">@T["The priority of this link to be displayed as active instead of another one when both are pointing to the same url. The link with the highest value has the active status in the navigation."]</span>
```